tools/update_deleted_packages.sh

```diff
@@ -90,6 +90,9 @@ fi
 # Find the child packages
 pkgs=( $(. "${find_pkgs_script}" --workspace "${workspace_root}") )
 
+# If no pkgs, then exit gracefully
+[[ ${#pkgs[@]} -gt 0 ]] || exit 0
```

Comment:
I think I'd put this in an if explicitly, but that's my `go` showing.

```bash
if [[ ${#pkgs[@]} -eq 0 ]]; then
  exit 0
fi
```

which feels a bit more simple to parse (YMMV).

Reply:
I had auto-merge on. I will update this in a follow-on PR.